plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/session/Session.kt

```diff
@@ -28,7 +28,7 @@ class Session(val tabID: String, val project: Project) {
     private var _conversationId: String? = null
     private var _latestMessage: String = ""
     private var task: String = ""
-    private val proxyClient: FeatureDevClient
+    var proxyClient: FeatureDevClient
```

Comment:
this does not need to be made public

just use `project.replaceService` to stub the client for tests